hasura.planx.uk/migrations/default/1748596202549_run_sql_migration/up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE VIEW "public"."planx_website_stats" AS
```

Comment:
We'll see if this holds up as a plain view! 

If it's slow / expensive we could...
 - Change to a materialised view, set up pg_cron to refresh daily
 - Proxy via the API and cache
 - Setup cache in webflow (no idea if possible yet!)